WORKSPACE

```diff
@@ -120,3 +120,16 @@ http_archive(
         "https://github.com/googleapis/gapic-showcase/archive/%s.zip" % _showcase_commit,
     ],
 )
+
+http_archive(
+    name = "rules_pkg",
+    sha256 = "8a298e832762eda1830597d64fe7db58178aa84cd5926d76d5b744d6558941c2",
+    urls = [
+        "https://mirror.bazel.build/github.com/bazelbuild/rules_pkg/releases/download/0.7.0/rules_pkg-0.7.0.tar.gz",
+        "https://github.com/bazelbuild/rules_pkg/releases/download/0.7.0/rules_pkg-0.7.0.tar.gz",
```

Comment:
I chose 0.7.0 to match the version in googleapis/googleapis here https://github.com/googleapis/googleapis/blob/ebf47e25ff363de57a6036562504db6caf3d8b89/WORKSPACE#L71